demisto_sdk/commands/common/tools.py

```diff
@@ -623,16 +623,16 @@ def get_script_or_integration_id(file_path):
         return commonfields.get('id', ['-', ])
 
 
-def get_api_module_integrations_set(changed_api_modules, integration_set):
+def get_api_module_integrations_set(changed_api_modules: Set, integration_set):
     integrations_set = list()
     for integration in integration_set:
         integration_data = list(integration.values())[0]
-        if integration_data.get('api_modules', '') in changed_api_modules:
+        if len(changed_api_modules.intersection(integration_data.get('api_modules', []))) > 0:
```

Comment:
```suggestion
        if changed_api_modules & integration_data.get('api_modules', ()):
```

Reply:
empty sets are falsey, so I think it should work